test/api/importcsv/api_test.go

```diff
@@ -0,0 +1,114 @@
+package importcsv
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+func TestImportCSV(t *testing.T) {
+	for _, r := range TestCases {
+		t.Run(r.FileName, func(t *testing.T) {
+
+			// Upload CSV.
+			inputData := make(map[string]interface{})
+			err := Client.FilePost("/importsummaries/upload", r.FileName, &inputData)
+			if err != nil {
+				t.Errorf(err.Error())
+			}
+
+			// Check list of Applications.
+			importedApps, _ := Application.List()
+			expectedApps := r.ExpectedApplications
+			for i, importedApp := range importedApps {
+				if i >= len(expectedApps) {
```

Comment:
I'd suggest check the len() of importedApps (or likely gotApps to be consistent with other tests) before the loop  to avoid running len(expected) in each step with always the same result. Loops with applications/dependencies checks might need a refactoring.